### DISCUSSION TITLE
Display relative dates / times

### DISCUSSION THREAD

**Discussion by sedubois** [avo-hq/avo#2541] [Feature requests]
I did not find an option in the [Date](https://docs.avohq.io/3.0/fields/date.html) and [DateTime](https://docs.avohq.io/3.0/fields/date_time.html) field to display them in relative terms, something like "1 day ago", etc. In Rails there is the `distance_of_time_in_words` utility for that I think. It would be nice to have that option.

**Comment by adrianthedev** [comment]
1. You can use a computed text field for that

```ruby
field :started_at_relative, as: :text do
  "#{time_ago_in_words(record.started_at)} ago"
end
```
2. We'll take a PR for an option like that in the date/date_time fields. Extra karma points for this!
